src/overrides.ts

```diff
@@ -0,0 +1,14 @@
+import * as vscode from 'vscode';
+
+const vscodeShowErrorMessage = vscode.window.showErrorMessage;
+
+async function calvaShowErrorMessage(message: string) {
+  const exlusionRegexps = [/Request textDocument\/codeAction failed./];
+  if (!exlusionRegexps.some((re) => re.test(message))) {
```

Comment:
We should probably only run exclusions when `document.languageId !== 'clojure'`.

Reply:
Anything coming through with that message is coming from clojure-lsp / Calva. I don't think this can have any effect on other extensions or other lsp servers, if that's what you're concerned about.

Reply:
With the above in mind, I don't think that check is necessary.

Reply:
That is not a safe assumption. CodeActions are part of LSP spec, and not limited to Clojure. While the odds of some other LSP is using the exact same error message is low, it is definitely not zero.

Reply:
Also the way this is written implies readiness to filter additional errors in the future. Without verifying Clojure, we need to be very careful and precise with future regex patterns.

Reply:
Maybe we should verify this, but I really don't think this would effect anything from any other extension. This override is specific to the runtime of the Calva extension alone.

@PEZ What's an easy way to test this? Maybe we could use a vsix from joyride that calls `showErrorMessage` and verify that they aren't intercepted by this override. I think that would be pretty crazy if they were.

Yes, this is written to make it easier to filter additional messages in the future if needed, however unlikely that may be.

Even if we verified the language is Clojure, if this override were to effect other extensions, it probably wouldn't be a good idea to use in the first place.

Reply:
I verified that this override does not affect calls to `showErroMessage` from other extensions by using a vsix of joyride that I made call that function, and a vsix of this PR that I made log when a message is intercepted.

Reply:
Awesome. Thanks for checking. And bigger thanks for finding and implementing this.

Reply:
To be honest, I haven't used joyride yet, and instead of learning the ropes I just did what I knew. 😄 But I maybe testing code that was compiled into the extension was a better test to be sure there's not some difference in how that command runs code? I don't know.

Reply:
That's good to know. I think we have some work to do with Joyride to relay just how easy it is to start using it. 😄 

If the test was to use `vscode/window.showErrorMessage` then this would have been tested just as well using Joyride. Everything that's done in a Joyride script is done in the context of the Joyride extension. FYI.

Reply:
Good to know! Thanks. Also, having not tried to get into Joyride yet, I don't know how easy it might be to get started, so I'm by no means suggesting it's not easy.